docs/spock_primer.adoc

```diff
@@ -599,19 +601,21 @@ activated by annotations called _directives_. Currently, Spock ships with the fo
 [horizontal]
 `@Timeout`:: Sets a timeout for execution of a feature or fixture method.
 
-`@Ignore`:: Ignores a feature method.
+`@Ignore`:: Ignores any feature method carrying this annotation.
 
-`@IgnoreRest`:: Ignores all feature methods not carrying this annotation. Useful for quickly running just a single method.
+`@IgnoreRest`:: Any feature method carrying this annotation will be executed, all others will be ignored. Useful
```

Comment:
Please set your IDE to softwraps instead of introducing needless newlines. Also fix the other additional line breaks.

Reply:
Change made to align with current line breaks.